docs/reference_guide.md

```diff
@@ -1847,6 +1849,42 @@ Examples in situ:
 [search /examples](https://github.com/iovisor/bcc/search?q=attach_xdp+path%3Aexamples+language%3Apython&type=Code),
 [search /tools](https://github.com/iovisor/bcc/search?q=attach_xdp+path%3Atools+language%3Apython&type=Code)
 
+### 10. attach_func()
+
+Syntax: ```BPF.attach_func(fn, attach_type [, attachable_fd [, flags)```
```

Comment:
I think we can have a signature like
```
BPF.attach_func(fn, attachable_fd, attach_type [, flags])
```
Basically make attachable_fd mandatory and flags optional for python APIs.
The only case that an attchable_fd is not needed (must be 0) is for net_namespace. I think it is okay to require it and this makes argument sequence consistent with C++ APIs.

Reply:
adjusted.